scripts/deploy-to-app-center.sh

```diff
@@ -35,10 +35,11 @@ appcenter distribute release \
 --app "$APP_CENTER_APP_NAME" \
 --release-notes-file "$TMP_DIR"/CHANGELOG.md \
 --group "$APP_CENTER_GROUP" \
---build-version "$APP_CENTER_BUILD_VERSION" \
+--build-version "$CIRCLE_BUILD_NUM" \
```

Comment:
do we need to modify the script? this will make it tied to Circle CI

Reply:
I was using `APP_CENTER_BUILD_VERSION`, but for some reasons, it wasn't showing in the Distribution groups. But now that we have a new group in AppCenter, I guess I can revert back this change